JDI.Light/JDI.Light/Factories/UIElementFactory.cs

```diff
@@ -40,11 +41,19 @@ public static IBaseUIElement CreateInstance(Type t, By locator, IBaseElement par
                         instance.Parent = parent;
                         return instance;
                     }
+
                 }
             }
             throw new MissingMethodException($"Can't find correct constructor to create instance of type {t}");
         }
-        
+
+
+        private static void DoNothing()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'DoNothing'.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5686813)